extension/manifest.json

```diff
@@ -8,9 +8,8 @@
     "128": "icon128.png"
   },
   "permissions": [
-    "https://accounts.atoma.cloud/*",
-    "https://bsp-td-prod.atoma.cloud/*",
-    "https://bsp-cdn-prod.atoma.cloud/*"
+    "alarms",
+    "*://accounts.atoma.cloud/*"
```

Comment:
Does this present as anything scary to users? like when the tabs permission requested browser history access

Reply:
@danreeves This is what it says - it shouldn't be nearly as scary to users and is better than the alternative
![image](https://user-images.githubusercontent.com/11822578/211360931-5f2b36dd-e46b-474d-a063-15fb62697899.png)